Week2/homework/Masoud/js-exercises/EX5-whoWantsADrink.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const drinkTray = [];
+const drinkTypes = ['cola', 'lemonade', 'water'];
+let num = 0;
+let courser = 0;
+
+for(let i = 0 ; i < 5 ; i++) {
+
+  drinkTray.push(drinkTypes[num]);
+  courser++;
+
+  if(courser == 2){
+    num++;
+    courser = 0;
+  }
+
+  if (num == drinkTypes.length){
+    num = 0;
+  }
+}
+
+console.log('Hey guys, I brought a ' + drinkTray + '!');
```

Comment:
This is wrong. The output would be different than the requirements. instead first convert the array to a string with commas (and spaces) then add it to the final string